nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/comparison/Eps.java

```diff
@@ -53,6 +53,7 @@ public Eps() {}
 
     public Eps(INDArray x, INDArray y, INDArray z) {
         super(x, y, z);
+        this.extraArgs = new Object[]{1e-2};
```

Comment:
Is 0.01 really a good default value for this? Typical epsilon values are 1e-6, 1e-7 or even 1e-12 for double.

Reply:
Fixed